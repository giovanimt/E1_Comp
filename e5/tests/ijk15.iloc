addI rsp, 4 => rsp
addI rsp, 4 => rsp
loadI 1 => r0
addI rfp, 0 => r1
store r0 => r1
loadI 234 => r2
addI rfp, 4 => r3
store r2 => r3
addI rfp, 0 => r4
load r4 => r5
loadI 0 => r6
cmp_GT r5, r6 => r7
cbr r7 => L0, L1
L0: nop 
loadI 393 => r8
addI rfp, 4 => r9
store r8 => r9
jumpI  => L2
L1: nop 
L2: nop 
