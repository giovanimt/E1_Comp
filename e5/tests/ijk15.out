00016878 1
00016882 393
